src/styles/_mixins.scss

```diff
@@ -5,16 +5,6 @@
 $global-left: if($global-text-direction == rtl, right, left);
 $global-right: if($global-text-direction == rtl, left, right);
 
-@mixin chevron($direction, $color) {
```

Comment:
I know we don't use it anywhere else at the moment but it might be a better way to draw chevrons than to use the SVGs as we're doing now in some components? Not saying to do all the changes now, but to keep the chevron generation as a mixin?